prism_compile.c

```diff
@@ -1074,6 +1074,19 @@ pm_scope_node_init(const pm_node_t *node, pm_scope_node_t *scope, pm_scope_node_
             scope->locals = cast->locals;
             break;
         }
+        case PM_ENSURE_NODE: {
+            pm_ensure_node_t *cast = (pm_ensure_node_t *)node;
+
+            // HAXXX
+            pm_constant_id_t err_info = 3863;
```

Comment:
We already have the top bit indicate that it's special. What do you think about:

| top bit | rest bits | result |
| --- | --- | --- |
| 0 | anything | actual constant |
| 1 | 0 | make a temporary |
| 1 | anything non-zero | mask off top bit, use rest as ID |

Is that too hacky?

Reply:
Looking at this again. I don't think we need this at all as we can just use `idERROR_INFO` I had overlooked this initially.